Dixit_Logic/Interfaces/IGameState.cs

```diff
@@ -91,5 +91,14 @@ IDeck BoardDeck
         {
             get;
         }
+
+        /// <summary>
+        /// Dixit game lasts until one of the players reach this point.
+        /// </summary>
+        int ObtainablePoint
```

Comment:
This is public, i cant add acces modifier here, only in the class what implement the interface (GameState)